README.md

```diff
@@ -24,3 +24,5 @@ An HTTP server with test methods for synthetic tests:
 * `GET /sleep?delay=5` will return a 204 after five seconds.
 * `GET /write-regularly?interval=1&count=10` will respond with 200 and write a 4 kB message into the response body every n seconds defined by the interval, n times defined by the count.
 * `/ws` endpoint serves a websocket with an echo service that return everything that is send to it.
+* `HEAD /filesystem?path=<path>`, returns 200 if the path exists and is a file, 204 if it exists but is a directory, 404 if not found, and 500 for any other error.
+* `GET /filesystem?path=<path>`, returns 200 with the file content if the path exists and is a file, 204 if it exists but is a directory, 404 if not found, and 500 for any other error.
```

Comment:
Nit-picking, I tried to have the list here sorted alphabetically by path. I acknowledge that there is already one place where it is incorrect, but if you prefer keeping things sorted as well, then feel free to adjust.

Reply:
Done